ts/src/binance.ts

```diff
@@ -3135,6 +3185,20 @@ export default class binance extends Exchange {
             }
         }
         const isMarginTradingAllowed = this.safeBool (market, 'isMarginTradingAllowed', false);
+        let marginMode = undefined;
+        if (spot) {
+            const hasCrossMargin = (id in this.options['crossMarginPairsData']);
+            const hasIsolatedMargin = (id in this.options['isolatedMarginPairsData']);
+            marginMode = {
+                'cross': hasCrossMargin,
+                'isolated': hasIsolatedMargin,
+            };
+        } else if (linear || inverse) {
+            marginMode = {
+                'cross': true,
+                'isolated': false,
```

Comment:
@ttodua what do you mean by false here?

Reply:
something mistake, i dont remember why i set false. fixing now